server/filters/renderChooseBarcodeOptionRadiosFilter.ts

```diff
@@ -7,25 +7,25 @@ export default function renderChooseBarcodeOptionRadiosFilter(values: Array<stri
       text: 'Print as part of your own document.',
       hint: {
         html: `        
-          <img src='/assets/images/print-barcodes-image.png' alt='Illustration showing the location of the barcode in the top left corner when inserted into a document' width='94px'/>
```

Comment:
I learned from the accessibility audit that as these are non-decorative images and already have instructional text beside them they should be ignored by screen readers - achieved with `alt=""`. Including an alt just doubled up on the instructions for each image which the tester found "frustrating and time consuming".